net/wireguard/src/opnsense/mvc/app/controllers/OPNsense/Wireguard/forms/dialogEditWireguardClient.xml

```diff
@@ -45,8 +45,8 @@
     </field>
     <field>
         <id>client.keepalive</id>
-        <label>Keepalive</label>
+        <label>Keepalive (seconds)</label>
```

Comment:
```suggestion
        <label>Keepalive Interval</label>
```

"(seconds)" looks lost here and explained in text as well. If this is a "visual" "problem" using interval would be a solution otherwise keeping the old label should be preferred over seconds addition